### ISSUE TITLE
[BUG] : reading from stdin in 0.1.1

### ISSUE THREAD

**Issue by SkwalExe** [SkwalExe/lsd-print#4]
Just found out reading from stdin (when no text is specified with `--`) isn't working properly, I'm working on it and will post updates soon.

Don't hesitate to open a pull request to solve this bug

**Comment by SkwalExe** [comment]
The problem is in 

```rust
if text.is_empty() {
                // if no text is directly given
                let stdin = io::stdin();
                let lines = stdin.lock().lines();
                println!("1");
                let mut line_count = 0;

                for line in lines {
                    println!("2");
                    // try to read from pipe
                    line_count += 1;
                    let line = line.expect("Could not read line from standard in");
                    drug_print(&unescape(&line).unwrap(), background.clone(), colors256);
                }

                println!("3");
                if line_count == 0 {
                    // if there is no text being piped into the program
                    println!("{}No text specified{}", RED, RESET);
                    process::exit(1);
                }
            }

            drug_print(
                &format!("{}{}", text, if newline { "\n" } else { " " }),
                background.clone(),
                colors256,
            );

            // just lsd-print the text
```

**Comment by SkwalExe** [comment]
found out what the problem was, It will be fixed soon

**Comment by SkwalExe** [comment]
The problem is fixed in 0.2.0 
see #5  for more information
